platform/osRegistryInterface.go

```diff
@@ -0,0 +1,46 @@
+//go:build windows
+// +build windows
```

Comment:
This module requires at least Go 1.18 to build, so we no longer need to add `// +build windows`. The `//go:build` pragma is the only one needed.